core/templates/pages/admin-page/roles-tab/admin-roles-tab.directive.ts

```diff
@@ -103,6 +103,7 @@ angular.module('oppia').directive('adminRolesTab', [
               ctrl.setStatusMessage('No results.');
             } else {
               ctrl.resultRolesVisible = true;
+              ctrl.contributionReviewersDataFetched = false;
```

Comment:
instead of binding these two different "forms" related variables, should we have separate variables to store the results? I think this will help us isolate the two functions and forms. wdyt?

Reply:
@DubeySandeep Yes, seems right.

Reply:
Done.